boot/bootutil/src/swap_move.c

```diff
@@ -449,6 +449,8 @@ swap_run(struct boot_loader_state *state, struct boot_status *bs,
     const struct flash_area *fap_sec;
     int rc;
 
+    BOOT_LOG_INF("Start images swap using SWAP_USING_MOVE algorithm.");
```

Comment:
Maybe shorten the "SWAP_USING_MOVE" to "MOVE' and "SWAP_USING_SCRATCH" to "SCRATCH", because the rest is in the log message anyway.

Reply:
I intended to use base-code terminology, but maybe this is bad here